rundeckapp/grails-app/migrations/core/NodeFilter.groovy

```diff
@@ -58,4 +58,23 @@ databaseChangeLog = {
             column(name: "project", type: '${varchar255.type}')
         }
     }
+
+    changeSet(author: "rundeckuser (generated)", failOnError:"false", id: "3.4.0-1616620097", dbms: "h2") {
```

Comment:
@imad6639 Is it okay to use timestamps in these IDs so the latest doesn't need to be hunted down through the files/db/etc? This is a pretty common db migration pattern for odering changes(timestamps in the names).